Documentation/community/roadmap.rst

```diff
@@ -0,0 +1,218 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Roadmap
+=======
+
+This page describes a high-level view of the main priorities for the project,
+agreed as a result of collaboration between Cilium's committers_ and the
+broader
+community. You'll also find here some pointers on how you can :ref:`influence
+the roadmap<rm-influence>`. 
+
+Major Feature Status
+--------------------
+
+
++-----------------------------+------------------------------------------------------------+
+| eBPF Networking             | Stable (:ref:`Roadmap Details<rm-advanced-networking>`)    |
+++----------------------------+------------------------------------------------------------+
+|| Kubernetes CNI             | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+|| Load Balancing             | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+|| Network Policy             | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+|| Kube-proxy Replacement     | Stable                                                     |
+++----------------------------+------------------------------------------------------------+
+| Multi-Cluster (ClusterMesh) | Stable (:ref:`Roadmap Details<rm-clustermesh>`)            |
++-----------------------------+------------------------------------------------------------+
+| Hubble Observability        | Stable (:ref:`Roadmap Details<rm-hubble-observability>`)   |
++-----------------------------+------------------------------------------------------------+
+| Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
++-----------------------------+------------------------------------------------------------+
+
+"Stable" means that the feature is in use in production (though advanced
+features may still be in beta or in development).
+
+Release Cadence
+~~~~~~~~~~~~~~~
+
+We aim to make 2-3 point releases per year of Cilium and its core components
+(Hubble, Cilium CLI etc). We also make patch releases available as necessary for
+security or urgent fixes. 
+
+Focus areas
+-----------
+
+For a finer-granularity view, and insight into detailed enhancements and fixes,
+please refer to `issues on GitHub <GitHub issues_>`_. 
```

Comment:
```suggestion
please refer to `issues on GitHub <GitHub issues>`_. 
```

Reply:
Although this one seems to be working fine as is, so not sure if this change is required or if I missed something.